src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.MemoryMappedInfo.cs

```diff
@@ -326,17 +304,12 @@ protected override void Dispose(bool disposing)
                 {
                     base.Dispose(disposing);
 
-                    if (_start != null)
+                    if (disposing)
                     {
-                        _accessor.SafeMemoryMappedViewHandle.ReleasePointer();
-                        _start = null;
+                        _handle.Dispose();
```

Comment:
➡️ Added a description to `MemoryMappedInfo` which describes these types as standard (totally non-special) implementations of `IDisposable` from the perspective of outside code, and linked to an article which covers many guidelines for implementing these types, including the reason why `_handle` is only disposed if `disposing` is true.